src/components/m-text-field/m-text-field.scss

```diff
@@ -0,0 +1,56 @@
+.m-text-field {
+  border: none;
+  display: inline-block;
+  position: relative;
+
+  &__field {
+    background-color: $background-light;
+    border: $default-border;
+    border-radius: $default-border-radius;
+    padding: $default-padding;
+
+    &:focus {
+      border-color: $focus-border-color;
+      box-shadow: 0 0 0 2px transparentize($focus-border-color, 0.8);
+      outline: none;
+    }
+
+    &:disabled {
+      cursor: not-allowed;
+    }
+
+    &--not-valid {
+      background-color: mix($error-border-color, $background-light, 10);
+      border-color: $error-border-color;
+
+      &:focus {
+        border-color: $error-border-color;
```

Comment:
To pensando se o padrão de borda e “outline” de erro deve sobrescrever o padrão de foco do input.

Reply:
Hum, eu dei uma olhada em alguns outros e vi que os mais bem resolvidos como o bootstrap fazem assim. E confesso que achei estranho ele ta com o fundo vermelho e a borda laranja hahaha

Reply:
Entendo. De qualquer forma ele tá colocando uma sombrinha quando foca, pra mostrar a diferenciação entre focado ou não, beleza então.